presto-benchto-benchmarks/pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <dep.logback.version>1.1.3</dep.logback.version>
+        <dep.logback.version>1.2.3</dep.logback.version>
```

Comment:
This version should be defined in the parent pom, so it is easy to keep it consistent across all the modules.